<map id="Mezzanine::NetworkURLException" name="Mezzanine::NetworkURLException">
<area shape="rect" id="node2" href="$classMezzanine_1_1NetworkException.html" title="Thrown when something unknown causes network IO to fail." alt="" coords="17,238,209,266"/>
<area shape="rect" id="node4" href="$classMezzanine_1_1IOException.html" title="Thrown when there was an issue with IO but very little was known about it." alt="" coords="35,161,192,189"/>
<area shape="rect" id="node6" href="$classMezzanine_1_1Exception.html" title="This is the exception thrown by most Mezzanine system that can throw exceptions." alt="" coords="41,83,185,111"/>
<area shape="rect" id="node8" title="STL class." alt="" coords="64,6,163,34"/>
</map>
